pkg/apis/common/resource.go

```diff
@@ -6,7 +6,7 @@ import (
 
 // PartialResource represents a linked resource
 type PartialResource struct {
-	Identifier string `json:"identifier,omitempty"`
+	Identifier string `json:"identifier,omitempty" anxcloud:"identifier"`
```

Comment:
PartialResource is not an Object and nothing should require this tag - I'll need more info what you tried to do and what happened please